### PR TITLE
add the third party FiPer

### PR DESCRIPTION
**Pull request by rodrigoodhin**

### CODE REVIEW

**Review by ReneWerner87 — APPROVED** [review]

**Comment by efectn** [comment]
Please check linter errors

File: README.md [hunk 1]
```diff
@@ -15,17 +15,20 @@ A curated list of awesome Fiber middlewares, boilerplates, recipes, articles and
...
+    - [🔗 External](#-external)
+    - [‍💻 Contrib](#-contrib)
+    - [🌱 Third Party](#-third-party)
+  - [🚧 Boilerplates](#-boilerplates)
+  - [📁 Recipes](#-recipes)
+  - [🛠️ Tools](#️-tools)
+  - [📖 Articles](#-articles)
+  - [📺 Videos](#-videos)
+  - [🤖 Benchmarks](#-benchmarks)
+    - [👍 Contributing](#-contributing)
```
Comment (efectn) [inline README.md]: No need to add contributing to here

Comment (rodrigoodhin) [reply README.md]: This one was added automatically. I've already removed it.

**Review by efectn — APPROVED** [review]

**Review by gaby — APPROVED** [review]

**Comment by rodrigoodhin** [comment]
Thank you @ReneWerner87 @efectn @gaby

**Review by ReneWerner87 — APPROVED** [review]

**Pull request merged**